components/server/src/workspace/workspace-starter.ts

```diff
@@ -1366,6 +1366,16 @@ export class WorkspaceStarter {
         let volumeSnapshotInfo = new VolumeSnapshotInfo();
         const volumeSnapshots = await this.workspaceDb.trace(traceCtx).findVolumeSnapshotById(volumeSnapshotId);
         if (volumeSnapshots !== undefined) {
+            log.info(
+                {
+                    lastInstanceId: lastValidWorkspaceInstanceId,
+                    volSnapshotId: volumeSnapshotId,
+                    snapshotId: volumeSnapshots.id,
+                    volumeHandle: volumeSnapshots.volumeHandle,
+                    workspaceId: workspace.id,
+                },
+                "starting workspace with volume snapshot info",
+            );
```

Comment:
would a metric be more useful than a log message or is there already something in place?

Reply:
No, we kind of need log instead for debugging reasons.